src/sass/veteran-representative.scss

```diff
@@ -0,0 +1,8 @@
+@import "shared-variables";
```

Comment:
You'll have to update this file to use the paths that match the other forms, which have changed recently (e.g. https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/burials/sass/burials.scss)